src/js/components/Tabs/doc.js

```diff
@@ -22,6 +22,12 @@ export const doc = Tabs => {
 This means that future tab changes will not work unless you subscribe to
 onActive function and update activeIndex accordingly.`,
     ),
+    alignTabs: PropTypes.oneOf([
```

Comment:
what about `alignControls` or something in that direction?

Reply:
I am fine with that.